src/simian/util/link_module.sh

```diff
@@ -21,6 +38,11 @@ function find_egg_file() {
   if [[ ! -z "${egg}" ]]; then
     echo "${egg}"
   fi
+
+  egg=$(find .eggs -type f -maxdepth 1 -name ${module}-*.egg 2>/dev/null)
```

Comment:
instead of two find blocks you could just add '.eggs' as a second argument to the find execution above

find . .eggs -type f ....